### ISSUE TITLE
Don't show "restored" entries for windows with no tabs, or windows with only blank tabs

### ISSUE THREAD

**Issue by cxw42** [cxw42/TabFern#186]
E.g.:

![image](https://user-images.githubusercontent.com/10515894/65432005-6c431e80-dde8-11e9-8ae4-53c4d9a8aeee.png)

The tab there was `chrome://newtab/`.  A window of only such tabs or `about:blank` tabs should not be shown as a "Recovered tabs" entry since there is nothing to recover!

**Comment by ahonn** [comment]
@cxw42 I can work on this

**Comment by cxw42** [comment]
Please see [updated contribution guidelines](https://github.com/cxw42/TabFern/blob/master/CONTRIBUTING.md#Legal).  Thanks!

**Comment by cxw42** [comment]
Note for future readers: the solution @ahonn provided was to skip saving the blank tabs rather than to adjust the way they are loaded.  That is not quite what I was thinking, but I believe it will have the desired result!

**Comment by cxw42** [comment]
@ahonn I am sorry to say I have just found a regression due to #191.  To reproduce:

- Open a Chrome window, and make sure it has an `about:blank` or `chrome://newtab` tab open.
- Make sure the TF window is open.
- Close the TF window.
- Click the TF toolbar button to re-open TF.

Expected: The TF tree shows one copy of the Chrome window

Observed: the TF tree shows two copies of the Chrome window, one as "Recovered tabs" without the blank/newtab tab.

![image](https://user-images.githubusercontent.com/10515894/66255577-2c1e5d00-e753-11e9-929f-5aba17c26cdf.png)

This happens because, when TF starts up, it checks each open window against the saved tab data.  If there is a match, it merges the "Recovered tabs" entry with the actual entry.  Since we are no longer saving the blank/newtab tabs, there is not a match, and so the merging fails.

#191 was a good-faith effort, so you still get Hacktoberfest credit for it :D .  I would be interested in a new PR to do the original work suggested by this issue: when the TF window opens, while populating the tree, don't populate a "Recovered tabs" entry that only includes blank/newtab tabs.  That would be here:

https://github.com/cxw42/TabFern/blob/9e5a64fe88929ec81c50ccccc67adce138774546/app/win/main_tl.js#L1634-L1639

Before calling `createNodeForClosedWindowV1`, or perhaps in `createNodeForClosedWindowV1`, check to make sure that:

- the window is not ephemeral; or
- there is at least one tab that is neither `about:blank` nor `chrome://newtab`.

Is that something you could work on?

**Comment by ahonn** [comment]
Yes, I can work on. And I open a new PR #193. Before calling `createNodeForClosedWindowV1 `, make sure the window not only includes `about:blank` or `chrome://newtab`
